src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -70,7 +71,17 @@ public override Task<ExportResult> ExportAsync(IEnumerable<Activity> activityBat
                         System.Console.WriteLine("Activity Tags");
                         foreach (var tag in activity.TagObjects)
                         {
-                            System.Console.WriteLine($"\t {tag.Key} : {tag.Value}");
+                            if (tag.Value.GetType().IsArray)
```

Comment:
not sure if the tag.Value.GetType() would throw NullRef.

Reply:
I would do:
```csharp
var array = tag.Value as IEnumerable;
if (array == null)
{
    // handle primitive types
    continue;
}
// handle array type

Reply:
thanks. i will change to that.

Reply:
testing locally, we can remove the `<object>` part and iterate it the same way.